api/controllers/todoListControllers.js

```diff
@@ -0,0 +1,59 @@
+'use strict';
+var request = require('request');
+var url = "https://api.createsend.com/api/v3.1";
+
+exports.listClients = function(req, res) {
+  request.get( {
+    url : url + "/clients.json",
+    headers : {
+      "Authorization" : "Basic " + new Buffer(req.headers.username).toString("base64")
+    }
+  }, function (error, response, body) {
+       res.json(body);
+     }); 
+};
+
+exports.createClient = function(req, res) {
+  request.post( {
+    url : url + "/clients.json",
+    json: req.body,
+    headers : {
+      "Authorization" : "Basic " + new Buffer(req.headers.username).toString("base64")
```

Comment:
wrap the common code of getting the default headers in a local method, and call that method everywhere. Do not repeat similar code over and over